bin/add-toc.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# ==============================================================================
+#/ This script recursively scans all markdown files in a given directory and
+#/ replaces all occurrences of "<!-- toc -->" with an actual table of content.
+#/
+#/ Requires the markdown-toc node module to be available in the project.
+# ------------------------------------------------------------------------------
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+shortUsage() {
+        echo ''
+        echo 'Usage: '
+        echo "         $0 <directory-to-scan>"
+        echo ''
+}
+
+fullUsage() {
+    grep '^#/' <"$0" | cut -c4-
+}
+
+add-toc() {
```

Comment:
As described above I noticed the following:
- There is no description above this function.